AssetRipper.Library/Exporters/Audio/AudioClipExporter.cs

```diff
@@ -14,7 +14,7 @@ namespace AssetRipper.Library.Exporters.Audio
 	public sealed class AudioClipExporter : BinaryAssetExporter
 	{
 		public AudioExportFormat AudioFormat { get; }
-		public AudioClipExporter(LibraryConfiguration configuration) => AudioFormat = configuration.AudioExportFormat;
+		public AudioClipExporter(LibraryConfiguration configuration) => AudioFormat = configuration.GetSetting<AudioExportFormat>();
```

Comment:
This method of accessing settings is more cumbersome and verbose.

Reply:
I think the small amount of extra characters is worth it for the benefits it provides, since more types can be added the the settings object without writing related code to drill the new settings down to wherever they're needed.